### PR TITLE
Fix enable accessibility

### PR DESCRIPTION
**Pull request by alschmiedt**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves
We were using both focusWorkspace and enableAccessibilityMode kind of sporadically to enter accessibility mode.

### Proposed Changes
We should have a single entry point for turning on and off accessibility mode. Enable accessibility mode now turns the Blockly.keyboardAccessibility flag to true and adds the cursor to either the selected block, the top block in the workspace, or the workspace itself.

### Test Coverage

Tested on:

### CODE REVIEW

File: core/keyboard_nav/navigation.js [hunk 1]
```diff
@@ -975,13 +977,15 @@ Blockly.navigation.toolboxOnAction_ = function(action) {
  */
 Blockly.navigation.enableKeyboardAccessibility = function() {
   Blockly.keyboardAccessibilityMode = true;
+  Blockly.navigation.focusWorkspace();
```
Comment (RoboErikG) [inline core/keyboard_nav/navigation.js]: is it an issue if this is called multiple times? Should it check the keyboardAccessibilityMode flag before making changes?

File: core/keyboard_nav/navigation.js [hunk 2]
```diff
@@ -975,13 +977,15 @@ Blockly.navigation.toolboxOnAction_ = function(action) {
...
 };
 
 /**
  * Disable accessibility mode.
  */
 Blockly.navigation.disableKeyboardAccessibility = function() {
   Blockly.keyboardAccessibilityMode = false;
```
Comment (RoboErikG) [inline core/keyboard_nav/navigation.js]: ditto

File: tests/mocha/navigation_test.js [hunk 3]
```diff
@@ -401,7 +401,7 @@ suite('Navigation', function() {
         Blockly.navigation.setCursor(this.workspace.cursor);
         Blockly.mainWorkspace = this.workspace;
         this.fieldBlock1 = this.workspace.newBlock('field_block');
-        Blockly.keyboardAccessibilityMode = true;
+        Blockly.keyboardAccessibility = true;
```
Comment (RoboErikG) [inline tests/mocha/navigation_test.js]: This variable doesn't exist anywhere else. Where did it come from?

File: tests/mocha/navigation_test.js [hunk 4]
```diff
@@ -633,10 +633,10 @@ suite('Navigation', function() {
       }]);
       var toolbox = document.getElementById('toolbox-categories');
       this.workspace = Blockly.inject('blocklyDiv', {toolbox: toolbox});
-      Blockly.navigation.focusWorkspace();
+      Blockly.navigation.enableKeyboardAccessibility();
       this.basicBlockA = this.workspace.newBlock('basic_block');
       this.basicBlockB = this.workspace.newBlock('basic_block');
-      Blockly.keyboardAccessibilityMode = true;
+      Blockly.navigation.enableKeyboardAccessibility();
```
Comment (RoboErikG) [inline tests/mocha/navigation_test.js]: Why is this called twice?

File: core/keyboard_nav/navigation.js (hunk 1 above)
Comment (alschmiedt) [reply core/keyboard_nav/navigation.js]: Neither of these should be a problem if it is called more than once, but I can add the check just to be safe.

File: tests/mocha/navigation_test.js (hunk 3 above)
Comment (alschmiedt) [reply tests/mocha/navigation_test.js]: thin air :/ should be fixed now

**Pull request merged**